agent-inject/agent/annotations.go

```diff
@@ -519,6 +525,14 @@ func Init(pod *corev1.Pod, cfg AgentConfig) error {
 		pod.ObjectMeta.Annotations[AnnotationTemplateConfigStaticSecretRenderInterval] = cfg.StaticSecretRenderInterval
 	}
 
+	//if maxConnsPerHostRaw, ok := pod.ObjectMeta.Annotations[AnnotationTemplateConfigMaxConnectionsPerHost]; !ok {
```

Comment:
Is this meant to be commented out?

Reply:
Nope, I forgot to add my actual changes. Whoops! Thank you.